src/main/java/com/typee/storage/Storage.java

```diff
@@ -7,12 +7,15 @@
 import com.typee.commons.exceptions.DataConversionException;
 import com.typee.model.ReadOnlyAddressBook;
 import com.typee.model.ReadOnlyUserPrefs;
+import com.typee.model.Tab;
 import com.typee.model.UserPrefs;
 
+import javafx.collections.ObservableList;
+
 /**
  * API of the Storage component
  */
-public interface Storage extends AddressBookStorage, UserPrefsStorage {
+public interface Storage extends AddressBookStorage, UserPrefsStorage, TypeeStorage {
```

Comment:
Can put TypeeStorage before UserPrefsStorage to preserve lexicographical ordering

Reply:
noted. will update in the next commit.